src/main/java/com/google/api/generator/engine/ast/AstNodeVisitor.java

```diff
@@ -50,6 +50,8 @@
 
   public void visit(JavaDocComment javaDocComment);
 
+  public void visit(MultiLineComment multiLineComment);
```

Comment:
Why do we need a new MultiLineComment? As discussed earlier today, we should reuse CommentStatement and add this to ClassDefinition.